migrations/versions/b8dea71fabf2_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
```

Comment:
Add a usable message here. I added comments for all the existing migrations in #36 if you'd like to see an example.